workbench/tests/renderer/invest_subprocess.test.js

```diff
@@ -0,0 +1,405 @@
+import fs from 'fs';
```

Comment:
Not much is new here, everything was just moved from `app.test.js`. Except the updates to `await userEvent` and to use `act` in a few places.